install_files/ansible-base/inventory-dynamic

```diff
@@ -37,16 +47,17 @@ def lookup_local_ipv4_address(hostname):
             monitor_ip = site_vars['monitor_ip']
 
     except (IOError, KeyError):
-        msg = "You must run 'secure-admin sdconfig' from root securedrop dir."
-        raise Exception(msg)
+        msg = ("Run `./securedrop-admin sdconfig` to configure"
```

Comment:
You are now missing the reference to running the command in the root directory. I don't see this as a huge-issue, but I thought my messaging before was adequate so I guess I have to agree to disagree here :) I'm not going to block though, I appreciate the other changes in this PR